app/assets/stylesheets/components/buttons.scss

```diff
@@ -410,3 +410,91 @@
     }
   }
 }
+
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+// .c-btn //////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+
+.c-btn {
```

Comment:
So this is something I kinda want to do more and more - we currently prefix all crayons-related classes with `crayons-` which is a bit long and make entire code look messy. So I thought about shortening it for future components and maybe slowly update that in existing ones.

Reply:
Just throwing it out there, but are we at a point where we could remove the prefix? Or not quite yet? I remember one reason was for handling some things in the CSS reset.

```sass
// TODO: Remove this once we enable box-sixing: border-box globally.
[class^='crayons'],
[class^='crayons']::before,
[class^='crayons']::after,
[class^='crayons'] *,
[class^='crayons'] *::before,
[class^='crayons'] *::after {
  box-sizing: border-box;
}
```

https://github.com/forem/forem/blob/main/app/assets/stylesheets/base/reset.scss#L1-L9

Reply:
Removing it at all? I'm not sure - do you think so? I kinda like having it because it helps keeping things a bit more unique and recognizable at the same time. 

- prefix does indicate that something is a component or at least from our codebase
- it helps avoiding potential conflicts with any 3rd party styling that could potentially affect our styles (simply by using very generic names that literally any library could affect, i.e. `.btn` or `.button` or `.form`.

But, in regards to that border-box from reset - yea, I think we could already drop this, just probably need to test it thoroughly first :)

Reply:
Sounds good for keeping the prefix.